src/serverdlg.cpp

```diff
@@ -331,7 +331,10 @@ lvwClients->setMinimumHeight ( 140 );
     tedWelcomeMessage->setText ( pServer->GetWelcomeMessage() );
 
     // prepare update check info label (invisible by default)
-    lblUpdateCheck->setText ( "<font color=\"red\"><b>" + QString ( APP_NAME ) + " " + tr ( "software upgrade available" ) + "</b></font>" );
+    lblUpdateCheck->setOpenExternalLinks ( true ); // enables opening a web browser if one clicks on a html link
+    lblUpdateCheck->setText (
+        "<font color=\"#c94a55\"><b>" + tr ( "A %1 upgrade is available:&nbsp;" ).arg ( APP_NAME ) +
+        tr ( "<a style='color:#c94a55;' href='https://jamulus.io/upgrade?progversion=%1'>Click for details</a>" ).arg ( VERSION ) + "</b></font>" );
```

Comment:
And why is this two separate strings, rather than a single string with two args?  I thought the style guide preferred the longer form so translators got more context?

Reply:
I'm not sure, but certainly for web site accessiblity you're supposed to put the context into the link text. So the text here would simply read "A Jamulus upgrade is availabe"  as a single link (or possibly "<a>A Jamulus upgrade</a> is available") without "Click for details", as decontextualised "click here" is annoying for screen readers. "Click for details" is also arguably redundant anyway.

Reply:
No, I really, _strongly_ disagree with relying on styling to indicate a clickable link.  Having the "click here" means you _know_ it's a link.  You're not left guessing as to what the styling might mean.

Reply:
I guess you don't want the formatting in the `tr( )` parts and, if you're going to have the HTML tags, it could be tricky.
```
setText ( QString ( tr ( "%1A %2 upgrade is available: %3Click for details%4" ) )
    .arg ( "<font color=\"red\"><b>" ).arg ( APP_NAME )
    .arg ( "<a style=\"color: red\" href=\https://jamulus.io/upgrade?progversion=" VERSION "\">" )
    .arg ( "</a></font>" ) )
```

Reply:
Fair enough. Links aren't conventional in desktop apps, where buttons are the standard. And the best practice for those is to put the action text on the button too (so in this case "update" or something). 

If we must go with a weblink, then I'm just pointing out the accessibility problem of 'click here' really. But I don't know enough about application screen readers to know.

https://webaccess.berkeley.edu/ask-pecan/click-here

Reply:
That would give
```
"%1A %2 upgrade is available: see %3details and downloads%4%5"
```
or something?

Reply:
I guess so, yes.

Reply:
I must have had a reason for this… the link should be translatable and e.g. in Spanish direct to jamulus.io/es/Upgrade. 
Concerning the HTML, yes that would work. 
I don’t have aaccess to my PC, so changes might need some time.

@gilgongo should we redirect /Upgrade to the latest release post or a already develop a separate "Upgrade" page?

Reply:
Do we have a "translators" code reviewers group that could be added to comment on things like this, too?  I'd like someone who's not too familiar with the code who does translations to give some perspective (rather than me guessing what's helpful).

Reply:
Kind of: @jamulussoftware/  trreviewers

Reply:
To me, something like
```
    lblUpdateCheck->setText ( tr ( "%1A %2 upgrade is available: go to %3https://jamulus.io/upgrade?progversion=%4details and downloads%5%6" ) )
        .arg ( "<font color=\"red\"><b>" ).arg ( APP_NAME )
        .arg ( "<a style=\"color: red\" href=\""  ) )
        .arg ( VERSION "\">" )
        .arg ( "</a>" ).arg ( "</b></font>" );
```

is overly complex.

Reply:
@jamulussoftware/trreviewers could you please comment on that? Do you care about HTML in your translations? Is this a problem?

Reply:
> should we redirect /Upgrade to the latest release post or a already develop a separate "Upgrade" page?

I think just redirecting /Upgrade to latest release will be fine for now.

Reply:
So that would be a manual process for each release (= https://jamulus.io/Upgrade would now link to https://github.com/jamulussoftware/jamulus/discussions/1813)

Reply:
> Do you care about HTML in your translations? Is this a problem?

It's not a problem either way AFAIC. Whenever there are formatting tags or content that doesn't need translation in a segment (which happens often) I just copy the source segment and overwrite the translatable content.

Reply:
Ok. So you‘d rather leave it like it is currently?

Reply:
I just mean that, as far as I'm concerned, it's not something that would bother me either way - I'd proceed without worrying about what translators might make of it.